src/ensembl/src/content/app/entity-viewer/state/general/entityViewerGeneralActions.ts

```diff
@@ -67,12 +66,8 @@ export const setDataFromUrl: ActionCreator<ThunkAction<
     }
   }
 
-  const entityId = params.entityId
-    ? buildEnsObjectId({
-        genomeId: genomeId as string,
-        ...parseFocusIdFromUrl(params.entityId)
-      })
-    : null;
+  const { entityId } = params;
```

Comment:
That can't be right.
`entityId` here is `gene:ENSG...`. The ids that we use for keys in redux store are `<genome_id>:<type>:<stable_id>`.

If you open the console, you will probably see the error:

![image](https://user-images.githubusercontent.com/6834224/83522378-7c171480-a4d8-11ea-8218-bae95d982b5b.png)

I believe the above lines, with `buildEnsObjectId`, should stay as they are.

Reply:
Okay, I thought the redux store should use the format `<type>:<stable_id>` since the sampleData was updated here:
https://github.com/Ensembl/ensembl-client/pull/282/files#diff-39dfc9125867911e956e3f1d875adf3eR873

Reply:
> I thought the redux store should use the format <type>:<stable_id> 

That won't be sufficient to distinguish between same types of entities with same ids but from different genomes (e.g. BRCA2 for grch37 and grch38); that's why I added genome id as a part of redux keys.

Reply:
> since the sampleData was updated here

That must have been my mistake. That PR went through several phases — at first I thought that we are changing everything to `<type>:<stable_id>`, but then realised that that won't work.

Sorry for forgetting to change the sampleData back.

Reply:
Thanks. I get it now. I have reverted the changes and updated the keys used in sampleData.